### PR TITLE
fix code style

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
     }
 }
 
-$json = json_encode($vars, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+$json = json_encode($vars, \JSON_PRETTY_PRINT | \JSON_UNESCAPED_SLASHES | \JSON_UNESCAPED_UNICODE);
```
Comment (derrabus) [inline src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php]: This code runs in the global namespace already. Why would we want backslashes here?

File: src/Symfony/Component/Validator/Tests/Fixtures/PropertyGetter.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
 /**
  * This class has two paths to PropertyGetterInterface:
  *   PropertyGetterInterface <- AbstractPropertyGetter <- PropertyGetter
- *   PropertyGetterInterface <- ChildGetterInterface <- PropertyGetter
+ *   PropertyGetterInterface <- ChildGetterInterface <- PropertyGetter.
```
Comment (derrabus) [inline src/Symfony/Component/Validator/Tests/Fixtures/PropertyGetter.php]: This does not look right.

Comment (xabbuh) [reply src/Symfony/Component/Validator/Tests/Fixtures/PropertyGetter.php]: reverted

File: src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php]: reverted

**Review by yceruto — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Pull request merged**